app/models/user_notification.rb

```diff
@@ -0,0 +1,20 @@
+class UserNotification < ActiveRecord::Base
+  include ApplicationNotificationsHelper
```

Comment:
> Main problem is that I need to get the email template path in send_email method.

Wait, the premise of your design is wrong.

Anything involving presentation should only affect the view, at most the controller. what does `add_template_helper` have anything to do with it?